cli/Cargo.toml

```diff
@@ -27,51 +27,59 @@ name = "fake-diff-editor"
 path = "testing/fake-diff-editor.rs"
 
 [build-dependencies]
-cargo_metadata = "0.17.0"
+cargo_metadata = { workspace = true }
 
 [dependencies]
-chrono = { version = "0.4.26", default-features = false, features = ["std", "clock"] }
-clap = { version = "4.3.19", features = ["derive", "deprecated"] }
-clap_complete = "4.3.2"
-clap_mangen = "0.2.10"
-config = { version = "0.13.3", default-features = false, features = ["toml"] }
-criterion = {version = "0.5.1", optional = true }
-crossterm = { version = "0.26", default-features = false }
-dirs = "5.0.1"
-esl01-renderdag = "0.3.0"
-git2 = "0.17.2"
-glob = "0.3.1"
-hex = "0.4.3"
-indexmap = "2.0.0"
-itertools = "0.11.0"
-jj-lib = { version = "=0.8.0", path = "../lib", default-features = false }
-maplit = "1.0.2"
-once_cell = "1.18.0"
-pest = "2.7.2"
-pest_derive = "2.7.2"
-regex = "1.9.1"
-rpassword = "7.2.0"
-serde = { version = "1.0", features = ["derive"] }
-slab = "0.4.8"
-tempfile = "3.7.0"
-textwrap = "0.16.0"
-thiserror = "1.0.44"
-timeago = { version = "0.4.1", default-features = false }
-toml_edit = { version = "0.19.14", features = ["serde"] }
-tracing = "0.1.37"
-tracing-chrome = "0.7.1"
-tracing-subscriber = { version = "0.3.17", default-features = false, features = ["std", "ansi", "env-filter", "fmt"] }
+chrono = { workspace = true, default-features = false, features = [
+    "std",
+    "clock",
+] }
```

Comment:
According to the doc, `features` are additive, and `default-features` can't be used, which means almost all inherited dependencies will be just `<name>.workspace = true`?

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace

Reply:
Yeah, I think so. I wasn't sure if I wanted that here or in another cleanup commit, but I think just doing it now makes the most sense.